src/screens/Login/Login.js

```diff
@@ -22,18 +29,30 @@ const Login = props => {
   const [emailError, setEmailError] = useState("");
   const [password, setPassword] = useState(passwordFromProps);
   const [passwordError, setPasswordError] = useState("");
+  const authService = new AuthService();
```

Comment:
As for me, we can use `AuthService` as a static class. What do you think?

Reply:
I tried to implement static methods in Auth class before, but the problem is to use inherited methods from Base class. For me there are two variant to avoid this issue - convert all methods in Base class into static or create instance of Auth class.